ansible/pulp-from-source.yml

```diff
@@ -14,6 +14,15 @@
       package:
         name: '*'
         state: latest
+    # FIXME(milan) Docker-provider specific issues
```

Comment:
This should probably be moved into a role.

Reply:
Cool, which role would you recommend?

Reply:
...moved to the devel role... I thought it was necessary/best to do in the pre-tasks because the ansible file module might require it, but we don't actually update any SELinux file attributes (as @Ichimonji10  points out), we just disable SELinux...